src/main/java/org/prebid/server/spring/config/PrometheusMapperConfiguration.java

```diff
@@ -0,0 +1,297 @@
+package org.prebid.server.spring.config;
+
+import io.prometheus.client.dropwizard.samplebuilder.CustomMappingSampleBuilder;
+import io.prometheus.client.dropwizard.samplebuilder.DefaultSampleBuilder;
+import io.prometheus.client.dropwizard.samplebuilder.MapperConfig;
+import io.prometheus.client.dropwizard.samplebuilder.SampleBuilder;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Configuration
+public class PrometheusMapperConfiguration {
+
+    @Bean
+    @ConditionalOnProperty(prefix = "metrics.prometheus.labels", name = "enabled",
+            havingValue = "false", matchIfMissing = true)
+    public SampleBuilder defaultSampleBuilder() {
+        return new DefaultSampleBuilder();
+    }
+
+    @Bean
+    @ConditionalOnProperty(prefix = "metrics.prometheus.labels", name = "enabled", havingValue = "true")
+    public SampleBuilder labelsSampleBuilder() {
+        final List<MapperConfig> mapperConfigs = new ArrayList<>();
+
+        mapperConfigs.addAll(accountModuleMapperConfigs());
+        mapperConfigs.addAll(moduleMapperConfigs());
+        mapperConfigs.addAll(accountMapperConfigs());
+        mapperConfigs.addAll(adapterMapperConfigs());
+        // other
+        mapperConfigs.add(requestsOk);
+        mapperConfigs.add(bidderCardinality);
+        mapperConfigs.addAll(privacyMappingConfigs());
+        mapperConfigs.addAll(vertxResponseConfig);
+        mapperConfigs.addAll(vertxRequestConfigs);
+        mapperConfigs.add(vertxServer);
+
+        return new CustomMappingSampleBuilder(mapperConfigs);
+    }
+
+    // example: requests.ok.openrtb2-web
+    private final MapperConfig requestsOk = new MapperConfig(
+            "requests.*.*",
+            "requests.status.type",
+            Map.of("status", "${0}", "type", "${1}")
+    );
+
+    // example: bidder-cardinality.4.requests
+    private final MapperConfig bidderCardinality = new MapperConfig(
+            "bidder-cardinality.*.requests",
+            "bidder-cardinality.requests",
+            Map.of("cardinality", "${0}"));
+
+    // example: vertx.http.clients.get-requests
+    // example: vertx.http.clients.post-requests
+    private final List<MapperConfig> vertxRequestConfigs = Stream.of(
+            "get", "post", "head", "delete", "connect", "options", "patch", "put", "other", "trace"
+    ).map(request -> new MapperConfig(
+                    "vertx.http.clients." + request + "-requests",
+                    "vertx.http.clients.requests_by_type",
+                    Map.of("request", request)
+            )
+
+    ).collect(Collectors.toList());
```

Comment:
with our code style it would be: 
```
Stream.of("get", "post", "head", "delete", "connect", "options", "patch", "put", "other", "trace")
    .map(request -> new MapperConfig(
                 "vertx.http.clients." + request + "-requests",
                  "vertx.http.clients.requests_by_type",
                  Map.of("request", request)))
    .collect(Collectors.toList());
```

Also, this should be a bean provider method, as i said before:
 ```
@Bean
    public List<MapperConfig> vertxRequestMapperConfigs() {
        return Stream.of("get", "post", "head", "delete", "connect", "options", "patch", "put", "other", "trace")
                .map(request -> new MapperConfig(
                        "vertx.http.clients." + request + "-requests",
                        "vertx.http.clients.requests_by_type",
                        Map.of("request", request)))
                .collect(Collectors.toList());
    }
```
Please, refactor other mappings to follow this pattern.